scrapy/extensions/feedexport.py

```diff
@@ -146,6 +146,32 @@ def _store_in_thread(self, file):
             key.close()
 
 
+class GCSFeedStorage(BlockingFeedStorage):
+
+    def __init__(self, uri, project_id, acl):
+        self.project_id = project_id
+        self.acl = acl
+        u = urlparse(uri)
+        self.bucket_name = u.hostname
+        self.blob_name = u.path[1:]  # remove first "/"
+
+    @classmethod
+    def from_crawler(cls, crawler, uri):
+        return cls(
+            uri,
+            crawler.settings['GCS_PROJECT_ID'],
+            crawler.settings['FEED_STORAGE_GCS_ACL']
+        )
+
+    def _store_in_thread(self, file):
+        file.seek(0)
+        from google.cloud.storage import Client
+        client = Client(project=self.project_id)
```

Comment:
You can also create the client using:

```python
json_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../google-cloud-service-account.json')
client = Client.from_service_account_json(json_path)
```

Reply:
Yup, this was in my original approach.
But, I kept compatibility with IMAGES_STORE for now.